components/sidebar/SidebarElement.vue

```diff
@@ -2,15 +2,24 @@
 import { version } from "~/package.json";
 
 const { data: collections } = usePrivatePlaylists();
+const isElectron = window.isElectron || false;
 </script>
 
 <template>
   <aside
     class="flex-none flex max-h-screen w-[300px] flex-col border-r border-label-separator bg-background-2"
   >
-    <div class="flex items-center p-3">
-      <SiteLogo size="small" />
+    <div
+      class="flex items-center p-3"
+      style="
+        -webkit-app-region: drag;
+        -webkit-user-select: none;
+        user-select: none;
```

Comment:
Why have those 3 CSS options been added to a couple of places? Shouldn't `user-select: none` be applied to the all the page? And what is it about `-webkit-app-region: drag`?

Reply:
Currently, you can drag the window using the titlebar. When the titlebar is hidden you have no way of moving the window. Therefore I defined the AppToolbar and the logo area as draggable so that you can move the window.

It might be a good idea to apply user-select: none everywhere, but I'm not sure if that messes with anything. Also, we probably shouldn't do it on the website.

Reply:
I'm now using user-select for the whole app if it's on Electron.